tools/vulnerability-scan/wss-scan.sh

```diff
@@ -47,33 +47,70 @@ basepath=$baseDirPath"/repos"
 
 echo "Cleaning up scan directories if already present"
 rm -rf $basepath
+echo "Cleaning up temp file that will affect scan"
+rm -rf /tmp/ws*
 
 mkdir -p $basepath
  
 # clone the desired Repos for scanning 
 for repo in $gitRepos
 do
   echo "Cloning repo "$gitBasePath$repo
-  git clone "$gitBasePath$repo".git $basepath"/"$repo
+  if [ ${repo} == "OpenSearch" ]
+  then
+    echo "Cloning "$repo" at branch 1.x"
+    git clone -b 1.x "$gitBasePath$repo".git ${basepath}/${repo}
```

Comment:
Temp measure to only scan 1.x for opensearch.
We need to add a new config option to specify repo version.
Ideally from the input menifest file of the releases.

Reply:
Temporarily we hardcoded this to clone 1.x branch only for OpenSearch. We plan to add more configuration options so that we could specify branch to scan for all repos.